src/ansys/dpf/post/static_mechanical_simulation.py

```diff
@@ -0,0 +1,4614 @@
+"""Module containing the ``StaticMechanicalSimulation`` class."""
+from typing import List, Tuple, Union
+import warnings
+
```

Comment:
I believe this file was copied from somewhere else?? 


Hence I'm not looking at it.